src/Api/Controllers/CollectionsController.cs

```diff
@@ -555,47 +555,45 @@ private async Task<CollectionResponseModel> Get_vNext(Guid collectionId)
     private async Task<CollectionAccessDetailsResponseModel> GetDetails_vNext(Guid id)
     {
         // New flexible collections logic
-        var (collection, access) = await _collectionRepository.GetByIdWithAccessAsync(id);
-        var authorized = (await _authorizationService.AuthorizeAsync(User, collection, BulkCollectionOperations.ReadWithAccess)).Succeeded;
+        var collectionDetails =
```

Comment:
The different models are a little hard to keep track of, so I suggest being verbose with the names - `collectionAdminDetails` here.

Reply:
Good call, fixed in [2f18bc8](https://github.com/bitwarden/server/pull/3793/commits/2f18bc8c0bd1e07cb8fbc30dcda537381996e351)